analyzers/tests/SonarAnalyzer.Test/TestCases/SpecifyRouteAttribute.CSharp12.cs

```diff
@@ -190,3 +190,23 @@ public class DerivedController : Controller { }
 
 [Controller]
 public class Endpoint { }
+
+public class ControllerWithInheritedRoute : ControllerWithRouteAttribute            // Compliant, attribute is inherited
+{
+    [HttpGet("Test")]
+    public string Index() => "Hi!";
+}
+
+[Route("api/[controller]")]
+public class ControllerWithRouteAttribute : Controller { }
+
+public class ControllerOverridesActionWithRoute : BaseControllerWithActionWithRoute // Noncompliant
+{
+    public override string Index() => "Hi!";                                        // Secondary
```

Comment:
This is a bit tricky. The method itself does not have the attribute. At the same time, the overridden method could be part of an API.

Reply:
Have you tested how overridden methods behave in an actual MVC application? Does MVC respect the Inherited flag?

Reply:
Thanks for documenting the routes!